RFC-001-BAM.md

```diff
@@ -369,7 +362,7 @@ Let's start off with an example:
 ```
 
 In the above example:
-- "Source-Ledger" is the header-key.
+- "source_ledger" is the header-key.
```

Comment:
❓Any particular reason why you are using double quotes here?
```suggestion
- `source_ledger` is the header-key.
```

Reply:
Nope. Does the rest of the document use back-ticks?